### ISSUE TITLE
New update should not offer builds from another release after selecting the first build

### ISSUE THREAD

**Issue by hroncok** [fedora-infra/bodhi#3648]
When I enter new update via https://bodhi.stg.fedoraproject.org/updates/new it offers me a list of builds to pick from. When I pick first, it should only offer me builds from the same release, as mixed updates are not possible.

**Comment by ryanlerch** [comment]
Bodhi actually has a feature here that if submitting a new update, if you list builds from multiple releases, it creates multiple updates for each release.

**Comment by hroncok** [comment]
Ohooo. That's super cool and super nonobvious. Maybe display an info box about this when user selects more releases?

**Comment by ryanlerch** [comment]
> Ohooo. That's super cool and super nonobvious. Maybe display an info box about this when user selects more releases?

Yeah, it apparently has been a secret feature for a looooong time :) 

Agreed that a subtle notification here might be good too. (or even maybe grouping the builds in the list by release too...)
